server/data/restClient.ts

```diff
@@ -23,6 +23,14 @@ interface PostRequest {
   raw?: boolean
 }
 
+interface PutRequest {
```

Comment:
Actually not required at this stage but forgot to remove, next step will be a PUT to update the reservation and complete.